app/code/Magento/Quote/Model/QuoteManagement.php

```diff
@@ -179,7 +186,8 @@ public function __construct(
         \Magento\Checkout\Model\Session $checkoutSession,
         \Magento\Customer\Model\Session $customerSession,
         \Magento\Customer\Api\AccountManagementInterface $accountManagement,
-        \Magento\Quote\Model\QuoteFactory $quoteFactory
+        \Magento\Quote\Model\QuoteFactory $quoteFactory,
+        \Magento\Quote\Model\Quote\Validator\MinimumOrderAmount\ValidationMessage $minimumAmountErrorMessage
```

Comment:
Due to backwards compatibility policy we can not introduce a new required constructor argument as it may break any code which extends from this class.
Please change this argument to be optional and use ObjectManager to get the dependency, it will be refactored later when such changes are allowed.
Example:
```php
public function __construct(
    \Magento\Quote\Model\OldDependency $old,
    \Magento\Quote\Model\NewDependency $new = null
) {
    $this->old = $old;
    $this->new = $new ?: \Magento\Framework\App\ObjectManager::getInstance()
            ->get(\Magento\Quote\Model\NewDependency::class);
}
```